srtcore/core.cpp

```diff
@@ -10748,8 +10756,8 @@ int srt::CUDT::processConnectRequest(const sockaddr_any& addr, CPacket& packet)
         HLOGC(cnlog.Debug, log << "processConnectRequest: ... NOT. Rejecting because broken.");
         return m_RejectReason;
     }
-    size_t exp_len =
-        CHandShake::m_iContentSize; // When CHandShake::m_iContentSize is used in log, the file fails to link!
+    // When CHandShake::m_iContentSize is used in log, the file fails to link!
+    size_t exp_len = CHandShake::m_iContentSize;
```

Comment:
This can be fixed easier - in 10772 use `+CHandShake::m_iContentSize` instead of `exp_len`.

Reply:
As I can see, you wrote that exactly this (using the `CHandShake::m_iContentSize` in log) fails to link.

Reply:
You missed the `+` before the name. This is exactly what fixes the problem.